packages/jellyfish-transaction-builder/src/index.ts

```diff
@@ -17,6 +18,7 @@ export * from './txn/txn_builder_utxo'
 export * from './txn/txn_builder_account'
 export * from './txn/txn_builder_oracles'
 export * from './txn/txn_builder_loans'
+export * from './txn/txn_builder_vault'
```

Comment:
https://github.com/DeFiCh/wallet/blob/10e8d710a00f917b4079f92f505aa398cc99c255/mobile-app/app/screens/AppNavigator/screens/Loans/screens/ConfirmCreateVaultScreen.tsx#L245

hmm.. we need the `txn_builder_loans` support the removed vault functions as well but state @deprecated
to avoid duplicates.. can try export the `txn_builder_vault` to `txn_builder_loans`??

Reply:
Hmm,  in this case I don't think its really possible to export `txn_builder_vaults` from `txn_builder_loans`, considering the deprecated functions are methods of the txn_builder class. I think in this case this is a neccessary duplication.